.github/workflows/tests.yml

```diff
@@ -10,9 +10,9 @@ jobs:
         - ubuntu-latest
         - windows-latest
         python:
-        - "3.6"
```

Comment:
for whatever reason, I can't get the CI pipeline to work on Windows with Python 3.6 anymore